### DISCUSSION TITLE
Transcription + Translation from English into French.

### DISCUSSION THREAD

**Discussion by t0t061** [chidiwilliams/buzz#488] [Q&A]
Hello,
First of all, thank you for this great audio transcription project, which is really promising. Very good work!
The "transcription" part doesn't pose any problems (apart from a few bugs / crashes, perhaps linked to my ageing PC running Win10 22H2), but I don't understand how the "translation" function works: I can transcribe "FR" audio to a "EN" text translation, but I can't do it the other way round, starting from an "EN" audio transcription to obtain a "FR" text translation. The text result is always in English, whatever the settings (Whipser / Medium / Translate / French).
Is this feature not yet available in this direction, or am I doing it wrong?
Thank you in advance for your help.

![image](https://github.com/chidiwilliams/buzz/assets/136443808/0976c9e4-7c77-41a1-bb05-3afc8bfbc1ca)

**Comment by chidiwilliams** [comment]
Hi, yes, that's correct. Whisper (and, therefore, Buzz) only supports X-to-X and X-to-English transcriptions, but not English-to-X. English-to-X models might be available sometime in the future, but they aren't for now.

**Reply by t0t061** [reply]
Thank you for this reply and clarification. We can only hope that the Whisper project will evolve in this direction in the future.

**Comment by t0t061** [comment]
However, I've just come across this discussion on the Whisper project which indicates the opposite: https://github.com/openai/whisper/discussions/649#discussioncomment-4693715
In fact, while the "translate" function allows you to translate any audio/video source language into English text, the "transcribe" function does the opposite. I started with 2 English audio files and ended up with a French text version.

![image](https://github.com/chidiwilliams/buzz/assets/136443808/f05e6f40-dcf2-4bf4-a009-b4ca22e5a29d)

I'm attaching the audio source file (ZIP) and the resulting text file to this message.
[Version Test.zip](https://github.com/chidiwilliams/buzz/files/11756237/Version.Test.zip)
[Version Test (Transcribed on 15-Jun-2023 12-17-52).txt](https://github.com/chidiwilliams/buzz/files/11756241/Version.Test.Transcribed.on.15-Jun-2023.12-17-52.txt)

I was pleasantly surprised!
